src/pages/JavaThreadDumpAnalyzer.jsx

```
@@ -0,0 +1,246 @@
+        <h1 className="text-3xl mb-6">Java Thread Dump Analyzer</h1>
+
+        <div
+          {...getRootProps()}
+          className={`border-2 border-dashed p-4 rounded-lg text-center cursor-pointer ${isDragActive ? 'border-blue-500 bg-blue-50' : 'border-gray-300'
+            }`}
+        >
+          <input {...getInputProps()} />
+          {isDragActive ? (
+            <p>Drop the thread dump here ...</p>
+          ) : (
+            <p>Drag 'n' drop file here, or click to select a file</p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)